core/drivers/atmel_wdt.c

```diff
@@ -240,33 +242,50 @@ static TEE_Result wdt_node_probe(const void *fdt, int node,
 	if (!wdt)
 		return TEE_ERROR_OUT_OF_MEMORY;
 
+	it_hdlr = calloc(1, sizeof(*it_hdlr));
```

Comment:
Can we have this block just above the `ITR_HANDLER()` line below?

Reply:
fair